app/src/displays/formatted-value/index.ts

```diff
@@ -13,7 +14,16 @@ export default defineDisplay({
 	handler: (value, options) => {
 		const prefix = options.prefix ?? '';
 		const suffix = options.suffix ?? '';
-		const formattedValue = options.format ? formatTitle(value) : value;
+
+		let sanitizedValue = String(value);
+
+		// Strip out all HTML tags
+		sanitizedValue = value.replace(/(<([^>]+)>)/gi, '');
```

Comment:
This regex seems very broad to me 🤔 are there situations where this could filter out legitimate content?

Reply:
Good question! I was lifting it from the existing displayValue of formatted-value so I'm not aware of whether it currently has false-positives as well 🤔

https://github.com/directus/directus/blob/efa64dc4aa7f631d9eb6ee942bc5302667b35ea2/app/src/displays/formatted-value/formatted-value.vue#L154-L158

Maybe we can use the `sanitize-html` package to be more thorough? Seems like we're already using it over in the API:

https://github.com/directus/directus/blob/efa64dc4aa7f631d9eb6ee942bc5302667b35ea2/api/src/utils/md.ts#L2

so perhaps we can add it to the App for this "strip HTML" purpose? 🤔 and set `allowedTags` to an empty array so that it'll actually strip all tags as [described by the package author here](https://stackoverflow.com/a/31639380).

Reply:
Maybe too much of an edge case this one, but as this sanitization is used for WYSIWYG html where if someone uses the `<` character in text it will match that and remove all text after that in the element
![image](https://user-images.githubusercontent.com/9389634/176189412-24259775-0e74-4821-9013-c0feb2f6a8d6.png)
```js
import sanitizeHTML from 'sanitize-html'; 
```
sounds like a good solution to me :) with all weird quirks of html/xml you really don't want to be matching that using regex

Reply:
Just noticed that `sanitize-html` doesn't bundle for browsers, so ended up using `dompurify` since it's already being used App side in the md utils anyway:

https://github.com/directus/directus/blob/52d1080c11de2e24432bf56ea03049533dc4503f/app/src/utils/md.ts#L2

uses `ALLOWED_TAGS: []` option similar to `allowedTags` in sanitize-html.